server/events.py

```diff
@@ -117,6 +105,10 @@ class ErrorData(EventData):
     message: str
 
 
+class SendBugsData(EventData):
+    """The data of a bugs introduction event."""
+
+
```

Comment:
Does this not need to be added to the EventRequest, or is the backend sending this exclusively as an EventResponse?

Reply:
Good catch! Maybe it wasn't giving me problems because the data is empty? :thinking: 

For now, I've implemented it as if the client has the timer and sends the bugs request (which might be a "security" problem in the future, since the user can technically avoid requesting bugs).